react/components/admin/pages/Form/Form.tsx

```diff
@@ -220,6 +228,15 @@ const Form: React.FunctionComponent<Props> = ({
         </>
       )}
       <FormFieldSeparator />
+      <Input
```

Comment:
I noticed the SEO Description and Keywords inputs were grouped inside the `isCustomPage` block and the new input for Robots is outside. Considering they're all related to SEO I was assuming they would stick together. That isn't the case?
https://github.com/vtex-apps/admin-pages/blob/58dde40617b973bae5df40c875a2e4081bb8b090/react/components/admin/pages/Form/Form.tsx#L203-L208

Reply:
This isCustomPage is for dynamic routes, it's not related to SEO, but, it does make sense when thinking that search pages and pre-defined routes like for example orderplaced, my account, and login. Since those pages have a different context and I only make it available for custom pages, it does make sense, but, maybe we should think about those routes in the future 🤔